tests/functests/MultipleLogManagersTests.cpp

```diff
@@ -36,13 +36,39 @@
 using namespace testing;
 using namespace MAT;
 
-class MultipleLogManagersTests : public ::testing::Test,
-                                 public HttpServer::Callback
+class RequestHandler : public HttpServer::Callback 
+{
+   public:
+    RequestHandler(int id) : count_(0), id_(id){}
+
+    int onHttpRequest(HttpServer::Request const& request, HttpServer::Response& response) override
+    {
+        UNREFERENCED_PARAMETER(response);
+        UNREFERENCED_PARAMETER(request);
+        std::string expected_url = "/" + std::to_string(id_) + "/";
+        EXPECT_EQ(request.uri, expected_url);
+        count_++;
+        return 200;
+    }
+
+    size_t GetRequestCount() {
+        return count_;
+    }
+
+   private:
+    size_t count_;
```

Comment:
Nit: Do we need the `_` suffix? Maybe just name these as `m_count` and `m_id` instead?

Reply:
Yes, I got used to follow the Google Style guide for [variable naming](https://google.github.io/styleguide/cppguide.html?msclkid=6e031e1dd15111ec940468bd031e25d6#Variable_Names) - we follow this in OpenTelemetry.
```
Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore.
```

But realized we use `m_` for members in 1DS. fixed it now.